### PR TITLE
Threepio: Support 1 -> many command translation

### PR DESCRIPTION
**Pull request by Nolski**
## Description
Threepio `0.0.10` now supports 1 -> many command translation. For commands such as `torch.nn.functional.linear` where no exact command exists in tensorflow.js, threepio will now return a plan-like object back which will consist of multiple commands that will result in the same result (for `linear` it will translate to `transpose` -> `matmul`).

## How has this been tested?
- Unit test in plan translation tests this.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by Nolski** [comment]
I'm not sure why the tutorial tests are failing here. The error log doesn't seem to point to any of the parts I've changed either.
